.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release
+on:
+  release:
+    types: [published]
+
+jobs:
+  release_to_npm:
+    name: Release to npm
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: 14.x
+      - run: npm install
+      - run: npm run build
+      - run: npm test
+      - run: npm run lint
+      - run: npm run cs-check
+      - run: npx lerna publish from-package --yes
```

Comment:
I use `from-package` instead of `from-git` so that it's possible to tag non-version-bump commits as new versions and still have the pipeline run properly.